src/Common/Core/Impl/Logging/Implementation/ServiceLogger.cs

```diff
@@ -7,12 +7,16 @@
 using Microsoft.Common.Core.Disposables;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.R.Host.UserProfile {
-    internal sealed class ServiceLogger : ILogger, IDisposable {
+namespace Microsoft.Common.Core.Logging {
+    public sealed class ServiceLogger : ILogger, IDisposable {
```

Comment:
Probably we should call it `EventLogLogger`, cause it is in `Common.Core` now and can be reused in code that isn't related to service.